docs/apps/components/cascader.md

```diff
@@ -0,0 +1,135 @@
+# Cascader
+
+## Description
+
+A **Cascader**, also known as a **Cascade Select** or **Dependent Drop-down**, is a type of input field in a form builder that allows users to select options dynamically based on their previous selections in other fields.
```

Comment:
Form Builder need to be changed to Page Builder to maintain consistency.

Reply:
done